client/my-sites/domains/domain-management/primary-domain/index.jsx

```diff
@@ -95,34 +95,38 @@ class PrimaryDomain extends React.Component {
 		const primaryDomainSupportUrl = SETTING_PRIMARY_DOMAIN;
 
 		return (
+			/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Instead of disabling each occurrence of that eslint rule let's do it only once at the top of the file to keep the code a bit cleaner. Adding `/* eslint-disable wpcalypso/jsx-classname-namespace */` to the top should do it.

Reply:
Thanks for the tip here!